docker/irods_catalog_provider/setup-4.3.4.input

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
consider using the unattended install rather than an input file...

https://docs.irods.org/4.3.4/getting_started/installation/#unattended-install